nmigen_cocotb.py

```diff
@@ -3,12 +3,50 @@
 from nmigen import Fragment
 from nmigen.back import verilog
 from nmigen.cli import main_parser, main_runner
+from nmigen.build.plat import TemplatedPlatform
 import subprocess
 import tempfile
 import os
 import shutil
 import inspect
 
+
+class TestPlatform(TemplatedPlatform):
+    device = 'test-device'
+    package = '000'
+    default_clk = None
+    resources = []
+    connectors = []
+
+    command_templates = ''
+    file_templates = ''
+    required_tools = ''
+    toolchain = ''
+
+    include_sim_files = True
```

Comment:
Where is it used? Why is it needed?

Reply:
ugly way of "telling" the core that sim files should be included. It's gone now